guides/shop-managers-guide/73-setup-taxes.md

```diff
@@ -0,0 +1,20 @@
+# Taxes
+
+To setup Taxes go to the admin and select Settings in the side bar and select Taxes. 
+![Admin Tax Settings](_assets/73-admin-taxes-find-settings.png)
+
+You will find two sections, Shop Tax Settings and Custom Tax Rates.
+![Shop Tax Settings](_assets/73-admin-taxes-shop-tax-settings.png)
+Under Shop Tax Settings you can primarly choose to have taxes setup or not. Under Primary tax service you can choose between None and Custom Rates. Only Custom Rates enables Fallback tax service. Default tax code for products applies to all products. The tax codes can be created in the Custom Tax Rates table below. Finally, click Save changes to save the settings.
+
+![Custom Tax Rates](_assets/73-admin-taxes-custom-tax-rates.png)
+To create a Custom Tax Rates click on the New Tax Rates button. This pops up a dialog with the following fields.
+![New Tax Rate](_assets/73-admin-taxes-new-tax-rate.png)
+- Rate: The percentage tax that needs to be applied.
+- Country: The origin or destination country for which the tax rates apply.
```

Comment:
I think we can make it more clear that people can set tax rates at whatever level they want, e.g. at the country level, the state level or down the city and zip code.

Reply:
Rephrased:-
Sourcing: You can pick between Origin and Destination. This allows different tax rates to be calculated based on the country, state or even ZIP of the sender or receiver.

Reply:
So this is actually incorrect. Selecting between Origin and Destination is unrelated to being able to set codes at any geographic level. Probably want to note that first.

Reply:
So the primary thing is that the way it's written now it seems like you should fill out Country, City, etc. when really you should first choose at what level you are going to set tax rates (at the country/state/city/zipcode level) and then create one tax rate for each one of these. Choosing Origin and Destination just determines which address is used for tax calculation.

Reply:
Added a note to explain the above comment. The tax rate creation form has a note explaining about the blank field rule.
![image](https://user-images.githubusercontent.com/75854210/147978684-64df8582-568e-4df9-be4d-7b95fcf54a1f.png)
The combined notes should convey the message.